lib/router.js

```diff
@@ -421,8 +421,9 @@ Router.prototype._handlePaths = function(rootNode, spec, scope) {
 
     // Handle paths
     // Sequence the build process with `.each` to avoid race conditions
-    // while building the tree.
-    return P.each(Object.keys(paths), function(pathPattern) {
+    // while building the tree. Also sort(), so that fixed path segments sort
+    // before patterns (by virtue of `[a-zA-Z/] < '{'`).
+    return P.each(Object.keys(paths).sort(), function(pathPattern) {
```

Comment:
This changes the order of the endpoints in the swagger-ui too. Right now the order is kinda 'logical', so that similar stuff is grouped together within a tag/category.